patches/series

```diff
@@ -1,5 +1,4 @@
-# This series applies on GIT commit v2.9.0
+# This series applies on GIT commit 8cadc751ca6cccbf2b8edbab6b11c01b347418eb
```

Comment:
Is there an associated semver with that SHA, or is this considered to be more precise?

Reply:
This is more precise